sa_ufal-ud-test.conllu

```diff
@@ -12,14 +12,14 @@
 # newpar id = panc0.p2
 # sent_id = panc0.s2
 # text = ओं नमः श्रीशारदागणपतिगुरुभ्यः।
-# translit = oṁ namaḥ śrīśāradāgaṇapatigurubhyaḥ.
-1	ओं	ओं	INTJ	_	_	2	discourse	_	Translit=oṁ|LTranslit=oṁ|Gloss=oh
-2	नमः	नमस्	NOUN	_	Case=Nom|Gender=Neut|Number=Sing	0	root	_	Translit=namaḥ|LTranslit=namas|Gloss=glory
-3-6	श्रीशारदागणपतिगुरुभ्यः	_	_	_	_	_	_	_	SpaceAfter=No|Translit=śrīśāradāgaṇapatigurubhyah|Gloss=(to-the-lords-honorable-Śāradā-and-Gaṇapati)
+# translit = oṁ namaḥ śrīśāradāgaṇapatigurubhyaḥ.
```

Comment:
Why have you changed the encoding of the accented characters in the transliteration?